File: A-vs-An/AvsAn-Python/AvsAn_test.py

from AvsAn import AvsAn

result = AvsAn.getInstance().query("")
print result
result = AvsAn.getInstance().query("x")
print result
result = AvsAn.getInstance().query("\"x")
print result
result = AvsAn.getInstance().query("unanticipated result")
print result
result = AvsAn.getInstance().query("unanimous vote")
print result
result = AvsAn.getInstance().query("honest decision")
print result

File: A-vs-An-Elixir/test/avs_an_test.exs
defmodule AvsAnTest do
  use ExUnit.Case

  test "article_for/1" do
    assert AvsAn.article_for("unanticipated result") == "an"
    assert AvsAn.article_for("unanimous vote") == "a"
    assert AvsAn.article_for("honest decision") == "an"
  end
end
